instrumentation/netty/netty-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_0/AttributeKeys.java

```diff
@@ -43,7 +43,7 @@
    * while the Attribute class is loaded by a third class loader and used internally for the
    * cassandra driver.
    */
-  private static <T> AttributeKey<T> attributeKey(String key) {
+  public static <T> AttributeKey<T> attributeKey(String key) {
```

Comment:
can you add a comment about why it's public? I still have same concern as https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1951/files#r546868569

Reply:
+1 I have added a comment there